easybuild/easyconfigs/f/FSL/FSL-5.0.10_makefile_fixes.patch

```diff
@@ -0,0 +1,276 @@
+# This fixes the build system of FSL. It now (mostly) uses EB.
+# Ward Poelmans <wpoely86@gmail.com>
+diff -ur fsl.orig/config/common/vars.mk fsl/config/common/vars.mk
+--- fsl.orig/config/common/vars.mk	2016-05-24 17:44:11.000000000 +0200
++++ fsl/config/common/vars.mk	2017-05-23 22:14:37.138797842 +0200
+@@ -24,14 +24,14 @@
+ USRCFLAGS =
+ USRCXXFLAGS =
+
+-LDFLAGS = ${ARCHLDFLAGS} ${USRLDFLAGS} -L. -L${DEVLIBDIR} -L${LIBDIR}
++LDFLAGS = ${EBVARLDFLAGS} ${ARCHLDFLAGS} ${USRLDFLAGS} -L. -L${DEVLIBDIR} -L${LIBDIR}
```

Comment:
not using them breaks the build system for me. Switching from `=` to `:=` breaks it.